instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -132,7 +132,7 @@ def client_response_hook(span: Span, message: dict):
 single item list containing all the header values.
```

Comment:
Note in the line above this one that it still says it replaces `-` with `_`. We should leave "(lowercase)" only.

Reply:
good point, fixed.